src/families/ripple/bridge/js.js

```diff
@@ -183,10 +183,16 @@ const broadcast = async ({ signedOperation: { signature, operation } }) => {
   const api = apiForEndpointConfig(RippleAPI);
   try {
     await api.connect();
-    const submittedPayment = await api.submit(signature);
+    const submittedPayment = await api.request("submit", {
+      tx_blob: signature,
+      fail_hard: false,
```

Comment:
Isn't that the default? https://xrpl.org/submit.html#main_content_body

Reply:
Yes is it